cache/contenthash/checksum.go

```diff
@@ -1073,7 +1074,13 @@ func (cc *cacheContext) scanPath(ctx context.Context, m *mount, p string, follow
 			txn.Insert(k, cr)
 		}
 		return nil
+	}
+
+	privileges := []string{winio.SeBackupPrivilege}
+	err = winio.RunWithPrivileges(privileges, func() error {
```

Comment:
this will need to be a platform specific file `checksum_windows.go`. See how I approached it in https://github.com/moby/buildkit/pull/4994